app.py

```diff
@@ -203,6 +221,68 @@ def get_current_image(prompt_type: PromptType) -> str:
         return DEFAULT_IMAGE_URL
 
 
+
+def get_current_holidays() -> list[dict[str, str]]:
+    """
+    Get the current holidays from the database and parse them into a structured list.
+    Returns a list of dicts: [{'name': 'Holiday Name', 'description': 'Description'}]
+    """
```

Comment:
While I think this is a good start, it's a very brittle approach. We should instead standardize the response using the `response_format` param to strictly return JSON and adapt the image prompt generation to use it instead of a text blob.